src/pages/en/guides/migrate-to-astro/from-nextjs.mdx

```diff
@@ -29,16 +33,825 @@ When you rebuild your Next.js site in Astro, you will notice some important diff
+    )
+}
+
+export const getStaticProps = async () => {
+    const res = await fetch("https://pokeapi.co/api/v2/pokemon?limit=151")
+    const resJson = await res.json();
+    const pokemons = resJson.results.map(pokemon => {
+        const name = pokemon.name;
+        // https://pokeapi.co/api/v2/pokemon/1/
+        const url = pokemon.url;
+        const id = url.split("/")[url.split("/").length - 2];
+        return {
+            name,
+            url,
+            id
+        }
+    });
+    return {
+        props: {
+            pokemons,
+        },
+    }
+}
+```
+
+#### Create `src/pages/index.astro`
+
+Use the return value of the Next function. Convert any Next or React syntax to Astro, including changing the case of any [HTML global attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes).
+
+Note that:
+
+- `.map` just works!
+
+- `className` becomes `class`.
+
+- `<Link>` becomes `<a>`.
+
+- The `<> </>` fragment is not required in Astro templating.
+
+```astro title="src/pages/index.astro" "class" "</a>" "<a"
+---
+---
+<ul class="plain-list pokeList">
+    {pokemons.map((pokemon) => (
+        <li class="pokemonListItem" key={pokemon.name}>
+            <a class="pokemonContainer" href={`/pokemon/${pokemon.name}`}>
+                <p class="pokemonId">No. {pokemon.id}</p>
+                <img class="pokemonImage" src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${pokemon.id}.png`} alt={`${pokemon.name} picture`}/>
+                <h2 class="pokemonName">{pokemon.name}</h2>
+            </a>
+        </li>
+    ))}
+</ul>
+```
+
+#### Add any needed imports, props and JavaScript
+
+Note that:
+
+- the `getStaticProps` function is no longer needed. Data from the API is fetched directly in the code fence.
+- The `<Layout>` component is imported, and wraps the page templating.
+
+```astro ins={2,4-16,19,31} title="src/pages/index.astro"
+---
+import Layout from '../layouts/layout.astro';
+
+const res = await fetch("https://pokeapi.co/api/v2/pokemon?limit=151");
+const resJson = await res.json();
+const pokemons = resJson.results.map(pokemon => {
+    const name = pokemon.name;
+    // https://pokeapi.co/api/v2/pokemon/1/
+    const url = pokemon.url;
+    const id = url.split("/")[url.split("/").length - 2];
+    return {
+        name,
+        url,
+        id
+    }
+});
+---
+
+<Layout>
+  <ul class="plain-list pokeList">
+      {pokemons.map((pokemon) => (
+          <li class="pokemonListItem" key={pokemon.name}>
+              <a class="pokemonContainer" href={`/pokemon/${pokemon.name}`}>
+                  <p class="pokemonId">No. {pokemon.id}</p>
+                  <img class="pokemonImage" src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${pokemon.id}.png`} alt={`${pokemon.name} picture`}/>
+                  <h2 class="pokemonName">{pokemon.name}</h2>
+              </a>
+          </li>
+      ))}
+  </ul>
+</Layout>
+```
+
```

Comment:
Do we want to add a shoutout back to our `Layout` port? It might seem obvious to us writing this, but I think reading is often non-linear with these types of guides.

Something akin to:

```suggestion

:::tip
Remember that this `Layout.astro` file [was migrated from our Next file and contains our styling and other contents.](#next-base-layout-to-astro)
:::

```

Reply:
(See my inline suggestion above)